### PR TITLE
Add ERC165 support

### PR DESCRIPTION
**Pull request by hieronx**
When opening a pull request to submit a new EIP, please use the suggested template: https://github.com/ethereum/EIPs/blob/master/eip-template.md

We have a GitHub bot that automatically merges some PRs. It will merge yours immediately if certain criteria are met:

 - The PR edits only existing draft PRs.
 - The build passes.
 - Your GitHub username or email address is listed in the 'author' header of all affected PRs, inside <triangular brackets>.
 - If matching on email address, the email address is the one publicly listed on your GitHub profile.

### CODE REVIEW

File: EIPS/eip-7540.md [hunk 1]
```diff
@@ -238,6 +238,13 @@ MUST be emitted when a redemption Request is submitted using the `requestRedeem`
       type: uint256
 ```
 
+### ERC-165 support
```
Comment (Joeysantoro) [inline EIPS/eip-7540.md]: I think this will fail linting if the link isn't in the header

**Review by Joeysantoro — COMMENTED** [review]
You'll want to add 165 to the requires list too

**Review by Joeysantoro — COMMENTED** [review]
You'll want to add 165 to the requires list too

File: EIPS/eip-7540.md (hunk 1 above)
Comment (hieronx) [reply EIPS/eip-7540.md]: Updated!

**Review by vikramarun — APPROVED** [review]
Yep, 165 would be great

**Pull request merged**